linera-core/src/client.rs

```diff
@@ -203,14 +209,33 @@ pub enum MessagePolicy {
 }
```

Comment:
@afck The names of these policies are slightly misleading. Perhaps `AcceptWhenPossible`, `RejectOrIgnoreWhenPossible`, `IgnoreWhenPossible` (unless `Ignore` also ignores protected messages? but that won't go well with reconfiguration)

Reply:
Indeed, I think it does, and it shouldn't!
I created https://github.com/linera-io/linera-protocol/issues/2395 for this.

Reply:
What's the usecase for auto-ignore btw?

Reply:
Originally I felt it would be good to make it possible to run a CLI command without handling any messages, i.e. make a block with only an operation and nothing else. (So it's less the node service or faucet that I had in mind, and more a CLI command.)
But maybe that's actually not important? Happy to remove that mode.

Reply:
I think we should address this in a subsequent PR.